### ISSUE TITLE
 troubles with Oracle VM VirtualBox

### ISSUE THREAD

**Issue by ederenn** [golemfactory/clay#1467]
after successful complete Golem installation (Oracle Oracle VM VirtualBox included), Oracle VM VirtualBox exe file keeps poping up for no reason. 

no actions were taken, no icon was clicked, Oracle VM VirtualBox setup keeps appearing multiple times. It is already installed, you can run it, shortcuts are created but from time to time, exe file asks for permission to install/make some changes on computer.

tested on:
![typ kompa](https://user-images.githubusercontent.com/32061891/31399823-db80b4ec-aded-11e7-8a13-bb8d24a09ecd.png)

**Comment by badb** [comment]
How many times did it happen? Can you add screenshots? Did you agree to make changes?

**Comment by ederenn** [comment]
Around six times. I don't have screenshots, it looked like standard launch of installation - pop-up asking permission to make changes. At the beginning I disagreed around three times, made no difference, so I agreed. This made no difference as well, it popped up again. Last time it popped, I disagreed. 
Most probably it stopped popping up after running the application - clicking the shortcut and closing the app window right after.
